lib/class-vip-request-block.php

```diff
@@ -25,15 +25,24 @@ class VIP_Request_Block {
 	 * @param string $value target IP address to be blocked.
 	 */
 	public static function ip( string $value ) {
+		// Don't run validation if the passed value is not a valid IP
+		if ( ! filter_var( $value, FILTER_VALIDATE_IP ) ) {
+			return;
```

Comment:
```suggestion
			error_log( 'VIP Request Block: The value passed is not a correct IP address: ' . $value );
			return;
```

Reply:
I committed this suggestion and the linting fails due to using `error_log` in production. Not sure what to do because the same method is used below in the same file.